public/app/features/alerting/unified/home/Insights.tsx

```diff
@@ -102,14 +111,17 @@ function getMimirManagedRulesScenes() {
 }
 
 function getMimirManagedRulesPerGroupScenes() {
-  const ruleGroupHandler = new QueryVariable({
-    label: 'Rule Group',
-    name: 'rule_group',
-    datasource: cloudUsageDs,
-    query: 'label_values(grafanacloud_instance_rule_group_rules,rule_group)',
-  });
-
-  return new EmbeddedScene({
+  // const ruleGroupHandler = new QueryVariable({
+  //   label: 'Rule Group',
+  //   name: 'rule_group',
+  //   datasource: cloudUsageDs,
+  //   query: 'label_values(grafanacloud_instance_rule_group_rules,rule_group)',
+  // });
+
+  return new NestedScene({
+    title: 'Mimir-managed Rules - Per Rule Group',
+    canCollapse: true,
+    isCollapsed: true,
```

Comment:
How do you think I should handle adding query variables so that we can filter these scenes by `rule_group`? seems it's not supported in `NestedScenes` right?

Reply:
I need to add support for this, will try to do that in the next 1-2 days

Reply:
This adds support for variables and controls 
https://github.com/grafana/scenes/pull/335

Reply:
and fixes styling a bit